server/src/main/java/com/hedera/block/server/persistence/storage/write/BlockAsDirWriterBuilder.java

```diff
@@ -16,8 +16,9 @@
 
 package com.hedera.block.server.persistence.storage.write;
 
+import static com.hedera.block.common.utils.FileUtilities.DEFAULT_DIR_PERMISSIONS;
```

Comment:
We shouldn't be digging into commons for constants, any constants we need belong in this module.
That said, there should not be defaults for file or folder permissions.  Those need to be in configuration or nowhere.

Reply:
Please see my response for this in the above comments, for the `BlockAsDirWriterBuilder` and `BlockAsDirWriterBuilder`, it applies here as well.

Reply:
✅ Done.